.github/workflows/semver.yml

```diff
@@ -12,40 +12,39 @@ jobs:
     permissions:
       contents: write
 
-    outputs:
-      new_release_published: ${{ steps.release.outputs.new_release_published }}
-      new_release_major_version: ${{ steps.release.outputs.new_release_major_version }}
-      new_release_minor_version: ${{ steps.release.outputs.new_release_minor_version }}
-      new_release_patch_version: ${{ steps.release.outputs.new_release_patch_version }}
+    outputs:      
+      published: ${{ steps.release.outputs.published }}
+      published_version: ${{ steps.release.outputs.published_version }}
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         fetch-depth: 0
 
     - name: Python Semantic Release
       id: release
-      uses: aasseman/python-semantic-release@gh_outputs
+      uses: tumaysem/python-semantic-release@master
       with:
+        additional_options: --github
         github_token: ${{ secrets.GITHUB_TOKEN }}
   
   container_build:
     runs-on: ubuntu-latest
     needs: release
-    if: needs.release.outputs.new_release_published == 'true'
+    if: needs.release.outputs.published == 'True'
     permissions:
       packages: write
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         fetch-depth: 0
     - name: Build Image
       id: build-image
       uses: redhat-actions/buildah-build@v2
       with:
         image: autoagora
-        tags: latest v${{ needs.release.outputs.new_release_major_version }}.${{ needs.release.outputs.new_release_minor_version }}.${{ needs.release.outputs.new_release_patch_version }} v${{ needs.release.outputs.new_release_major_version }}.${{ needs.release.outputs.new_release_minor_version }} v${{ needs.release.outputs.new_release_major_version }}
```

Comment:
I would like to keep the multi-tier release tags such that when ie. v1.2.3 gets released, these tags get updated to the latest container hash:
- `v1`
- `v1.2`
- `v1.2.3`

As seen here: https://github.com/semiotic-ai/autoagora/pkgs/container/autoagora, or in general with most up to snuff popular containers out there.